docs/source/tls.rst

```diff
@@ -0,0 +1,58 @@
+.. _tls:
+
+
+TLS/SSL
+=======
+
+Currently dask distributed has experimental support for TLS/SSL communication,
+providing mutual authentication and encryption of communications between cluster
+endpoints (Clients, Schedulers and Workers).
```

Comment:
Is it worth noting auth+encrypt is the supported mode and they are not implemented separately?